cli/Application/Command/Install/Install.php

```diff
@@ -68,7 +68,7 @@ public function execute()
 			if (!$this->getApplication()->input->get('reinstall'))
 			{
 				$this->out()
-					->out('<fg=black;bg=yellow>' . g11n3t('WARNING: A database has been found !!') . '</fg=black;bg=yellow>')
+					->out('<fg=black;bg=yellow>' . g11n3t('WARNING: A database has been found!') . '</fg=black;bg=yellow>')
 					->out()
 					->out(g11n3t('Do you want to reinstall ?'))
```

Comment:
@brianteeman they call it [Plenken](https://en.wikipedia.org/wiki/Plenken) and I am very guilty of it... @MATsxm since we are now translating everything you may also want to remove my smileys (`=;)`) and other inconsistencies like `Finished` vs. `Finished.` that are used several times - but only if your time frame permits this - THANKS for your hard work `=;)`